net-misc/electrum/electrum-3.0.3.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI="6"
 
-PYTHON_COMPAT=( python3_{4,5} )
+PYTHON_COMPAT=( python3_{4,5,6} )
```

Comment:
this ebuild has a number of issues, like depending on naked `dev-lang/python`, which should use `${PYTHON_DEPS}` instead